express-spotify-app/views/home-random.hbs

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.6.2/css/bulma.css">
+  <title>PokeFun</title>
+</head>
+<body>
+ <section class="section">
+    <div class="container is-fluid">
+      <div class="notification">
+        <div class="media-content">
+            <div class="field">
+                <form class="control is-expanded" method="GET" action ="/artists">
+                  <input class="search" name="searchterm" type="text" placeholder="put a Artist">
+                </form>
+                <a href="./artists.hbs" class="button is-link is-medium is-primary is-outlined is-expanded">
```

Comment:
You don't send the user to a file, you send him to a url and then handle the request to that url with `app.get()`